src/main/java/edu/ucsb/cs56/ucsbapi/academics/curriculums/v1/classes/Course.java

```diff
@@ -43,6 +52,10 @@ public String getTitle() {
         return this.title;
     }
 
+    public String getTitleNoSpace() {
```

Comment:
It appears that getTitleNoSpace might not be used at the moment.   Consider removing this method if it isn't being used.

Reply:
updated!